src/data/modifiers.txt

```diff
@@ -8189,20 +8237,25 @@ Skill	Gift of the Maid	Meat Drop: +50
 Skill	Gift of the Priest	Spell Damage: +40
 Skill	Gift of the Scribe	Experience (Mysticality): +5
 Skill	Gluteus Maximus	Muscle: +25
+Skill	Gluttony	Experience: -2, Stomach Capacity: +2
+# Gluttony duplicates Opossum's food gains
 Skill	Gnefarious Pickpocketing	Meat Drop: +10
 Skill	Gnomish Hardigness	Maximum HP Percent: +5
 Skill	Golgi Apparatus	Absorb Stats: +25
 # Good Singing Voice: Doubles duration of Boris songs
 Skill	Gorgonzola's Guile	Item Drop: +25
 # Gourmand: Increases adventures from food
-# Gravitational Compression: +100% Item Drops from Monsters (Scales with your Mysticality)
 Skill	Gravitational Compression	Item Drop: [min(ceil(basemys/5),100)]
+# Gravitational Compression: +100% Item Drops from Monsters (Scales with your Mysticality)
+Skill	Greed	Meat Drop: +50, Item Drop: -15
+# Grogged For Battle
```

Comment:
a curiosity: why have some picked up the "+X Y Warfare Effectiveness" and some not?

Reply:
No idea! We should write some tests and see